includes/classes/admin/class-permalinks.php

```
@@ -163,4 +190,66 @@ public function apply_taxonomy_slugs($args, $taxonomy)
+	 */
+	public function get_post_type_fields( $fields ) {
+
+		global $CONTENT_MODEL_JSON_PATH;
+		if ( ! isset( $CONTENT_MODEL_JSON_PATH ) ) {
+			return $fields;
+		}
+		
+		foreach ( $CONTENT_MODEL_JSON_PATH as $json_path ) {
+			$types = glob( $json_path . '/post-types/*.json' );
+			$types = array_map(
+				fn( $file ) => json_decode( file_get_contents( $file ), true ),
```

PHPStan (2.1.46)

[warning] Parameter `#1` $json of function json_decode expects string, string|false given.


(argument.type)